src/test/java/hw/five/steps/ThenStepDef.java

```diff
@@ -0,0 +1,24 @@
+package hw.five.steps;
+
+import static org.testng.Assert.assertEquals;
+
+import io.cucumber.java.en.Then;
+
+public class ThenStepDef extends AbstractBaseStepDef {
+
+    @Then("{string} page should be opened")
+    public void pageShouldBeOpened(String pageName) {
+        assertEquals(driver.getTitle(),
+                pageName);
+    }
+
+    @Then("{int} log row has {string} text in log section")
+    public void logRowHasTextInLogSection(int firstRow, String logRow) {
+
+    }
```

Comment:
Empty method should be removed if it is never used

Reply:
method removed